package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@kiali/kiali-ui",
   "version": "1.18.0",
+  "proxy": "https://52.116.156.242:32263/kiali/",
```

Comment:
Ops, we don't commit proxies in package.json, unless is a change on the libraries or scripts this file should not be commited.